defaults/main.yml

```diff
@@ -22,18 +13,26 @@ react_git_key_filename: "id_ed25519"
 react_remove_git_key: true
 
 # system-wide dependencies
-react_system_wide_dependencies:
+react_remote_system_wide_dependencies:
   - yarn
+  - git
+react_local_system_wide_dependencies:
+  - git
 react_node_version: 10.x
 
 # app
+react_remote_js_build: True
+react_local_checkout_path: "/tmp/reveal"
```

Comment:
Is it possible to make this something like `/tmp/$USER/{ the react app name }`?

Reply:
yeah sure.